### DISCUSSION TITLE
Unable to teach-in an HmIP-eTRv-2 device

### DISCUSSION THREAD

**Discussion by ericleigh007** [OpenCCU/OpenCCU#2053] [🇺🇸 Q&A] [answered]
Hello everyone,

Hoping this is something easy.

I'm not able to "teach-in" certain HmIP devices with the raspberrymatic loaded on my pi3.

The homematic integration seems to work fine, I can control devices, and my heater boiler program runs great, but I cannot add any new devices by "teaching them in".

I've tried both automatic by removing the battery, by resetting the device, pushing various buttons, to no avail.  I also did the key/SGTIN mode and that didn't work either. (long-lived users may have seen me here on these boards with previous problems that turned out to be a homematic ip hub and the raspberrymatic running at the same time, but that's not it this time --  my hub is off and powered down and in a drawer -- somewhere).

The device I'm trying to teach-in is an e-TRv-2, ordered and received today from Conrad.

There are blurbs in the log when I turn on full debug mode, referring to "unsupported device" and "no device description found for manufacturer code"

I attached my log files in case that helps anyone.  I'm sort of surprised there is no indication in the log when the teach-in mode starts, which may have made things easier, but the device was at least seen.   The SGTIN ends in 349d.

-thanks
-e
[de838cd8-raspberrymatic-2022-11-17.log](https://github.com/jens-maus/RaspberryMatic/files/10026766/de838cd8-raspberrymatic-2022-11-17.log)

**Comment by jp112sdl** [comment, marked as answer]
Which version of RaspberryMatic are you running?

`No device description found for manufacturer code: 1 device type 484 sgtin 3014F711A000395F29A6349D `

The device type "484" is integrated since 3.65.8.xxxx

**Reply by ericleigh007** [reply]
Thanks!
It was easy (we'll find out after I upgrade)

I was so happy to finally get through the backup/restore process that I forgot all about upgrading things.

I hope I didn't waste too much of your time.  I'll come back to close this one later tonight after I get off work.

**Comment by ericleigh007** [comment]
Yes, works fine.  Thanks for all your help.  As always, a very helpful and friendly set of folks here.
Cheers.
